lib/block_view_post.go

```diff
@@ -801,9 +809,16 @@ func (bav *UtxoView) _connectSubmitPost(
 				PkToStringBoth(txn.PublicKey), spew.Sdump(GetParamUpdaterPublicKeys(blockHeight, bav.Params)))
 		}
 
-		// Modification of an NFT is not allowed (before the specified block height).
-		if existingPostEntryy.IsNFT && blockHeight < bav.Params.ForkHeights.AllowUpdatingNFTPostsBlockHeight {
-			return 0, 0, nil, errors.Wrapf(RuleErrorSubmitPostCannotUpdateNFT, "_connectSubmitPost: ")
+		if blockHeight >= bav.Params.ForkHeights.AllowUpdatingNFTPostsBlockHeight {
+			// Modification of a frozen post is not allowed after the above block height.
+			if existingPostEntryy.IsFrozen {
```

Comment:
we need to set `IsFrozen` on `newPostEntry` within the `if len(txMeta.PostHashToModify) != 0 {` statement. Currently I only see us setting `IsFrozen` on new posts